op-program/client/l2/engineapi/l2_engine_api.go

```diff
@@ -177,6 +177,31 @@ func (ea *L2EngineAPI) endBlock() (*types.Block, error) {
 }
 
 func (ea *L2EngineAPI) GetPayloadV1(ctx context.Context, payloadId eth.PayloadID) (*eth.ExecutionPayload, error) {
+	return ea.getPayload(ctx, payloadId)
+}
+
+func (ea *L2EngineAPI) GetPayloadV2(ctx context.Context, payloadId eth.PayloadID) (*eth.ExecutionPayloadEnvelope, error) {
+	payload, err := ea.getPayload(ctx, payloadId)
+	if err != nil {
+		return nil, err
+	} else {
```

Comment:
This doesn't necessarily need the `else` block but does the same thing, I think it reads better without it